app/utils.py

```diff
@@ -17,49 +18,39 @@ def __init__(self, bot_path, item_type):
             )
         )
         with open(config_file, "r") as file:
-            self.configs = yaml.safe_load(file)
+            configs = yaml.safe_load(file)
 
-        if self.configs is None:
+        if configs is None:
             raise ValueError("No config found in the file")
 
-        self.list = self.configs.get(item_type)
+        self.list = configs.get(self.item_type)
         if self.list is None:
-            raise ValueError(f"No {item_type} found in the config file")
-
-        for list_items in self.list:
-            if list_items.get("media") is None:
-                raise ValueError(
-                    f"No media found in the config file for {list_items} in {item_type}"
-                )
-            elif list_items.get("format") is None:
-                raise ValueError(
-                    f"No format found in the config file for {list_items} in {item_type}"
-                )
+            raise ValueError(f"No {self.item_type} found in the config file")
+
+        errors = [
```

Comment:
Can you enhance readability here? You can pull out the for loop and just keep adding a error-list imho.